docs/api_docs/python/conf.py

```diff
@@ -42,7 +42,7 @@
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store", "*.venv", "README.md"]
```

Comment:
Why `*.venv` instead of `.venv`?

Reply:
Matching our .gitignore: https://github.com/iree-org/iree/blob/41dcee93c7157955d94973addf6770cecf926849/.gitignore#L59-L62

(and I have multiple virtual environments like `3.11.venv`, `3.12.venv`, `iree.venv`, etc.)

Reply:
Okay, fine for me. (I normally have `.venv-${name}` to find them more easily 😉 )

Reply:
... that's a very good idea :O